Rakefile

```diff
@@ -40,7 +40,7 @@ namespace :test do
 end
 
 task :performance do
-  require 'analyzer'
+  require File.expand_path('~/Code/analyzer/lib/analyzer', __FILE__)
```

Comment:
I don't think this makes me able to run the benchmark...

Reply:
Whoops yea, I was playing around with sizing of the graphs; need to add an option to analyzer somehow to set the size; will remove